es6-arrow-functions/main.js

```diff
@@ -0,0 +1,45 @@
+const numbers = [12, 11, 10, 9, 8, 7, 6];
+const book = {
+  name: 'Bea Wolf',
+  author: 'Zach Weinersmith',
+  illustrator: 'Boulet',
+  published: 2023,
+};
+
+console.log('Exercise 1');
+const doubled = numbers.map((n) => {
+  return n * 2;
+});
```

Comment:
FYI 💡: We can take advantage of the syntax of arrow functions here when returning a simple expression and remove the block `{}` syntax: 

`const doubled = numbers.map((n) => n * 2);`